samples/Azure.ResourceManager.Sample/Generated/VirtualMachineExtensionVirtualMachineScaleSet.cs

```diff
@@ -11,13 +11,13 @@
 
 namespace Azure.ResourceManager.Sample
 {
-    /// <summary> A Class representing a VirtualMachineExtensionVirtualMachineScaleSets along with the instance operations that can be performed on it. </summary>
-    public class VirtualMachineExtensionVirtualMachineScaleSets : VirtualMachineExtensionVirtualMachineScaleSetsOperations
+    /// <summary> A Class representing a VirtualMachineExtensionVirtualMachineScaleSet along with the instance operations that can be performed on it. </summary>
```

Comment:
looks like you might need to sync the latest feature/v3

Reply:
Done